README.md

```
@@ -2,12 +2,45 @@
+## Options
+
+Here is a list of all available command-line flags:
+
+| Flag                  | Shorthand | Description                                                                 | Default |
+| --------------------- | --------- | --------------------------------------------------------------------------- | ------- |
+| `--volname`           |           | Set the volume name for the DMG.                                            | ""      |
+| `--disk-image-size`   |           | Set the size for the DMG in megabytes (MB).                                 | 0       |
+| `--codesign`          |           | Provide a signing identity to codesign the final DMG.                       | ""      |
+| `--notarize`          |           | Provide keychain-stored credentials to notarize the DMG.                    | ""      |
+| `--bless`             |           | Bless the disk image.                                                       | `false` |
+| `--apfs`              |           | Use APFS as the disk image's filesystem.                                    | `false` (HFS+) |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)